packages/components/select/src/select-button.ts

```diff
@@ -81,7 +76,9 @@ export class SelectButton extends ScopedElementsMixin(LitElement) {
     }
 
     return html`
-      <div>${selected || this.placeholder || '\u00a0'}</div>
+      <div id="button-content" class=${this.placeholder && !selected ? 'placeholder' : ''}>
```

Comment:
What's the DOM id for?

Reply:
It's used for the `aria-labelledby` in the `listbox`.

Reply:
But that has to cross the shadow DOM, right? `aria-labelledby` doesn't support that.

Reply:
Right, good one, I'll just add `aria-label` for the `listbox`.